### ISSUE TITLE
I can't use the `keys` function in V4

### ISSUE THREAD

**Issue by davetapley** [mikefarah/yq#665]
**Describe the bug**
I can't use the `keys` function in V4.

Version of yq: 4.3.2
Operating system: linux
Installed via: snap

**Command**
The command you ran:
```
yq e -n 'keys'
```

**Actual behavior**

```yaml
Error: Parsing expression: Lexer error: could not match text starting at 1:1 failing at 1:2.
        unmatched text: "k"
```

**Expected behavior**

```yaml
yq: error (at <unknown>): null (null) has no keys
```

**Additional context**

That expected behavior is from `jq-1.5-1-a5b5cbe`.

**Comment by mikefarah** [comment]
Changed to enhancement - `yq` doesn't yet have everything `jq` can do - but I'm working on it!

**Comment by mikefarah** [comment]
Fixed in 4.4.0
https://mikefarah.gitbook.io/yq/operators/keys
